uint8 role
uint8 id
float32 distance
float32 fpRssi
float32 rxRssi
uint32 systemTime
